spec/features/crop_searches_spec.rb

```diff
@@ -10,20 +10,21 @@ def host_with_port
   end
   
   it 'handles empty searches', js: true do
-    visit root_path
-    fill_in 'q', with: ''
-    FactoryGirl.create_list(:crop, 10)
-    Crop.searchkick_index.refresh
-    click_button 'Search'
-    title = Crop.first.name
-    #first test - look for title
-    expect(page).to have_content(title)
-    #first test - look for description
+    # visit root_path
```

Comment:
instead of commenting out, you could mark as `pending`

Reply:
Hi @Br3nda ..you mean pending as an issue label or..?

Reply:
i mean, replace the word `it` with `pending`.
The the specs you want to keep but haven't go passing yet, will still be run, but their failures won't cause the whole build to fail.

Reply:
Instead of this
```
  it 'handles empty searches', js: true do
    # visit root_path
    # fill_in 'q', with: ''
    # FactoryGirl.create_list(:crop, 10)
    # Crop.searchkick_index.refresh
    # click_button 'Search'
    # title = Crop.first.name
    # # Look for crop title
    # expect(page).to have_content(title)
    # # Look for crop description

    # description = Crop.first.description
    # expect(page).to have_content(title)
    # # Image fix to come after configuration is set up

    # image = Crop.first.main_image_path 

end
```

you can do this

```
  pending 'handles empty searches', js: true do
    visit root_path
    fill_in 'q', with: ''
    FactoryGirl.create_list(:crop, 10)
    Crop.searchkick_index.refresh
    click_button 'Search'
    title = Crop.first.name
    # Look for crop title
    expect(page).to have_content(title)
    # Look for crop description

    description = Crop.first.description
    expect(page).to have_content(title)
    # Image fix to come after configuration is set up

    image = Crop.first.main_image_path 

end
```

Reply:
If you comment it out, then rspec will output that testing "handles empty searches" passed, and future people will think that scenario is testing

If it's pending, then rspec will say "pending spec: handles empty searches" and still pass, but remind people in the future that this scenario is not tested and work is needed. A future contributor can look at the list of pending specs and find a place to work and contribute.